optimizer.py

```diff
@@ -65,3 +67,25 @@ def fitness(solution, solution_idx):
     print('Found solution with cost: ', plan.cost(), ' after ', fitness.cost_recalc_counter,
           ' cost recalculations.')
     return plan
+
+
+def bees_alg_opt(plan):
+    routers_cnt = len(plan.get_features()['routers'])
+    search_boundaries = ([0.0 for _ in range(routers_cnt)], [1.0 for _ in range(routers_cnt)])
+    cost_recalc_counter = 0
+
+    def hypersphere(solution):
+        nonlocal cost_recalc_counter
+        plan.set_routers_coeffs(solution)
+        cost_recalc_counter += 1
+        return -plan.cost()
+
+    alg = BeesAlgorithm(hypersphere, search_boundaries[0], search_boundaries[1])
+    alg.performFullOptimisation(max_iteration=5)
+    best = alg.best_solution
+    plan.set_routers_coeffs(best.values)
+    print("Bee algorithm:")
+    print("Parameters of the best solution : {solution}".format(solution=best.values))
+    print('Found solution with cost: ', plan.cost(), ' after ', cost_recalc_counter,
```

Comment:
Maybe think of some kind of logging? That'd be unified and easy to compare across algorithms